starter-code/src/main.js

```diff
@@ -1,50 +1,107 @@
+  { name: "captain america", img: "captain-america.jpg" },
+  { name: "fantastic four", img: "fantastic-four.jpg" },
+  { name: "flash", img: "flash.jpg" },
+  { name: "green arrow", img: "green-arrow.jpg" },
+  { name: "green lantern", img: "green-lantern.jpg" },
+  { name: "ironman", img: "ironman.jpg" },
+  { name: "spiderman", img: "spiderman.jpg" },
+  { name: "superman", img: "superman.jpg" },
+  { name: "the avengers", img: "the-avengers.jpg" },
+  { name: "thor", img: "thor.jpg" },
+  { name: "aquaman", img: "aquaman.jpg" },
+  { name: "batman", img: "batman.jpg" },
+  { name: "captain america", img: "captain-america.jpg" },
+  { name: "fantastic four", img: "fantastic-four.jpg" },
+  { name: "flash", img: "flash.jpg" },
+  { name: "green arrow", img: "green-arrow.jpg" },
+  { name: "green lantern", img: "green-lantern.jpg" },
+  { name: "ironman", img: "ironman.jpg" },
+  { name: "spiderman", img: "spiderman.jpg" },
+  { name: "superman", img: "superman.jpg" },
+  { name: "the avengers", img: "the-avengers.jpg" },
+  { name: "thor", img: "thor.jpg" }
 ];
 
-$(document).ready(function(){
+$(document).ready(function() {
   var memoryGame = new MemoryGame(cards);
-  var html = '';
-  memoryGame.cards.forEach(function (pic, index) {
+
+  //  shuffle
+  memoryGame.shuffleCard(cards);
+  // var counter = 0;
+  var html = "";
+  memoryGame.cards.forEach(function(pic, index) {
     html += '<div class= "card" id="card_' + pic.name + '">';
-    html += '<div class="back"';
-    html += '    name="'       + pic.img +  '">';
-    html += '</div>';
-    html += '<div class="front" ';
+    html += '<div class="back check1"';
+    html += '    name="' + pic.img + '">';
+    html += "</div>";
+    html += '<div class="front check2" ';
     html += 'style="background: url(img/' + pic.img + '") no-repeat"';
-    html += '</div>';
-    html += '</div>';
+    html += "</div>";
+    html += "</div>";
   });
 
   // Add all the div's to the HTML
-  document.getElementById('memory_board').innerHTML = html;
+  $("#memory_board").html(html);
   // Bind the click event of each element to a function
-$('.back').on('click', function () {
-   
-});
-});
 
+  $(".back").on("click", function() {
+    if (memoryGame.pickedCards.length !== 2) {
+      $(this).toggleClass("back");
+      $(this).toggleClass("front");
+      $(this)
+        .next()
+        .toggleClass("back");
+      $(this)
+        .next()
+        .toggleClass("front");
+      // counter += 1;
+      // console.log(counter);
+      memoryGame.pickedCards.push($(this).attr("name"));
+      // console.log(memoryGame.pickedCards.length);
+      setTimeout(function() {
+        if (memoryGame.pickedCards.length === 2) {
+          var check = memoryGame.checkIfPair(
+            memoryGame.pickedCards[0],
+            memoryGame.pickedCards[1]
+          );
+          var finish = memoryGame.finished();
+          // console.log(check);
+          if (check === false) {
```

Comment:
you could also simply write 
`if(!check)`
even better would be to name this variable something like `isPair` and then 
`if(!isPair)`